repo/content/content_manager_test.go

```diff
@@ -2403,6 +2405,13 @@ func (s *contentManagerSuite) newTestContentManager(t *testing.T, st blob.Storag
 	return s.newTestContentManagerWithTweaks(t, st, nil)
 }
 
+func (s *contentManagerSuite) newTestContentManagerWithMetadataCompression(t *testing.T, st blob.Storage, comp compression.Name) *WriteManager {
```

Comment:
Is `comp` used?

I wonder why it was not flagged by a linter. 🤔

Reply:
This function is not used anymore, I forgot to cleaned it up. I've removed this